bolt/lib/Target/X86/X86MCPlusBuilder.cpp

```diff
@@ -1876,6 +1876,13 @@ class X86MCPlusBuilder : public MCPlusBuilder {
     //    add %r2, %r1
     //    jmp *%r1
     //
+    // or a fixed indirect jump template:
+    //
+    //    movslq En(%rip), {%r2|%r1}
+    //    lea PIC_JUMP_TABLE(%rip), {%r1|%r2}     <- MemLocInstr
```

Comment:
Update this comment and the one above it to reflect the new matching logic: there is no MemLocInstr anymore

Reply:
Updated the comment. There's still MemLocInstr as a return argument.